src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar So, if you still want report the transcoding speed, here are 2 command lines:
`ffmpeg64.exe -y -i jellyfish-60-mbps-hd-hevc-10bit.mkv -c:v libx264 -tune fastdecode -preset superfast -crf 22 -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov -seekable 1 pipe: -benchmark`

`ffmpeg64.exe -y -i jellyfish-60-mbps-hd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 4M -c:v libx264 -tune fastdecode -preset superfast -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov -seekable 1 pipe: -benchmark`